src/main/webapp/i18n/en/bonus.json

```diff
@@ -0,0 +1,56 @@
+{
+    "artemisApp": {
+        "bonus": {
+            "bonusStrategy": {
+                "title": "Select a Bonus Strategy",
+                "helpText": "GRADES: First, calculates target exam grade. Then, applies bonus to that.<br>POINTS: First, applies bonus to the student's points. Then, calculates the final grade by matching the resulting points to the target exam's grading key.",
+                "grades": "GRADES",
+                "points": "POINTS"
+            },
+            "discreteness": {
+                "title": "Select discreteness",
+                "helpText": "DISCRETE: (Not available yet) Bumps the student's grade to a better grade step. Final grade will be one of the grade steps in the target exam (e.g. from 1.3 to 1.0 or from B to A). <br>CONTINUOUS: Applies bonus arithmetically to the student's grade. Final grade can be any numeric value between the best and the worst grade step values (e.g. from 1.3 to 1.2).",
+                "discrete": "DISCRETE",
+                "continuous": "CONTINUOUS"
+            },
+            "calculation": {
+                "title": "Select calculation",
+                "helpText": "− (Default option for GRADES): Subtracts bonus from target exam's grades/points. Prefer this when lower means better.<br>+ (Default option for POINTS): Adds bonus to target exam's grades/points. Prefer this when higher means better."
+            },
+            "bonusSource": {
+                "title": "Select bonus source",
+                "helpText": "Exams and Courses with \"Bonus\" type Grading Keys are listed below. The student's grade for the selected course or exam determines the bonus amount.",
```

Comment:
Is this correct?
```suggestion
                "helpText": "Exams and Courses with \"Bonus\" type Grading Keys are listed below. The student's points or grade for the selected course or exam determines the bonus amount.",
```

Reply:
Yes the initial version is correct because the numeric value of the `gradeName` you received from the bonus source course/exam is used in the calculation. Well, if you go one step deeper the points affect the `gradeName` you would receive so saying it won't be wrong but the initial version would be simpler in my opinion.